Source/options.cpp

```diff
@@ -1829,7 +1829,15 @@ std::vector<ModOptions::ModEntry> &ModOptions::GetModEntries()
 	if (modEntries)
 		return *modEntries;
 
-	std::vector<std::string> modNames = ini->getKeys(name);
+	std::vector<std::string> modNames = ini->getKeys(key);
+
+	// Add mods available by default:
+	for (const std::string_view modName : { "clock" }) {

```

Comment:
hopefully we can get this to scan the mod folder instead

Reply:
The built-in mods folder may be in devilutionx.mpq, which is tricky to scan